src/factories/routeFactory.ts

```diff
@@ -7,8 +7,22 @@ export interface IRouteResult{
     route?: IRoute;
 }
 
+interface IExternalRoute {
+    lintunnus: string;
+    reikuka: string;
+    reinimi: string;
+    reinimilyh: string;
+    reinimilyhr: string;
+    reinimir: string;
+    reitinsuuntasByReitunnus: {
+        edges: Object[],
```

Comment:
Would it be better if we used a slightly different query to get rid of `edges`

Reply:
Similar to how lineFactory can use `nodes`

Reply:
Also why Object and not IExternalRoutePath or something..

Reply:
Changed this, resolving